lua/lua/sipper.lua

```diff
@@ -44,7 +54,7 @@ function Sipper:verify_usip(tUsipConfigData, strComSipFilePath, strAppSipFilePat
             strCompareSipPath = strAppSipFilePath
             strCompSip = "APP"
         else
-            tResult = false
+            uResult = VERIFY_RESULT_ERROR
```

Comment:
The for loop continues with the next usip if an error/mismatch occurs. Is this intentional?